Pinta.Core/Actions/HelpActions.cs

```diff
@@ -47,10 +47,11 @@ public HelpActions ()
 			fact.Add ("Menu.Help.Translate.png", new Gtk.IconSet (PintaCore.Resources.GetIcon ("Menu.Help.Translate.png")));
 			fact.AddDefault ();
 
-			Website = new Gtk.Action ("Website", Catalog.GetString ("Pinta Website"), null, "Menu.Help.Website.png");
-			Bugs = new Gtk.Action ("Bugs", Catalog.GetString ("File a Bug"), null, "Menu.Help.Bug.png");
-			Translate = new Gtk.Action ("Translate", Catalog.GetString ("Translate This Application"), null, "Menu.Help.Translate.png");
-			About = new Gtk.Action ("About", Catalog.GetString ("About"), null, Stock.About);
+			// Translators: "Application name" (Pinta) + "Website"
```

Comment:
No problem, this isn't well-documented - I've added a wiki page (https://github.com/PintaProject/Pinta/wiki/Translations) about how translations work, which hopefully helps